assume/strategies/flexable_storage.py

```diff
@@ -130,7 +130,8 @@ def calculate_bids(
 
             # if price is higher than average price, discharge
             # if price is lower than average price, charge
-            if price_forecast[start] >= average_price:
+            # if price forecast favors discharge, but max discharge is zero, set a bid for charging
```

Comment:
Where is this implemented? Or is this outdated?

Reply:
This is implemented through the change in the line below:

`if price_forecast[start] >= average_price and max_power_discharge[start]:`

By adding the second clause, we only set a discharge bid, if we do have something to discharge ;)